src/test/resources/jsonld/HT020719239.json

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
actually, we don't need this test file, because HT015894164 already tests the morph.
I propose to get rid of my commit .

Reply:
But if HT015894164 is a needed test file later, it will show this change then additional to the "needed" change is should show. Therefore I would disagree.

Reply:
I agree, will do.

Reply:
+1